types/amphtml-validator/amphtml-validator-tests.ts

```diff
@@ -0,0 +1,11 @@
+import * as ampHtmlValidator from 'amphtml-validator'
+
+;(async () => {
+    const validator = await ampHtmlValidator.getInstance()
+    const result = validator.validateString('<html></html>')
```

Comment:
Tests should include a use of `result` too

Reply:
Added some usage of the `result` object and its members `status` and `errors` as well.